### PR TITLE
Create grading.json

### PR DESCRIPTION
**Pull request by blhohmann**
add grading.json

### CODE REVIEW

File: grading.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  “GROUP“ : “G001“,
+  “ALL_SPRINTS“ : {
+    “SPRINT1_URL“ : https://final-project-team-001-fabc72e5fdcb.herokuapp.com/,
```
Comment (evfinkn) [inline grading.json:4]: I don't think this should have a comma at the end right?

Comment (evfinkn) [reply grading.json:4]: Does it need quotes around the url?

**Review by evfinkn — APPROVED** [review]

File: grading.json (hunk 1 above)
Comment (blhohmann) [reply grading.json]: the picture doesn't have quotes around url, so I'm not sure
image

Comment (powersaudrey25) [reply grading.json]: i don't think there needs to be quotes

**Pull request merged**